crates/core/tedge_actors/src/errors.rs

```diff
@@ -39,6 +40,9 @@ pub enum RuntimeError {
 
     #[error(transparent)]
     LinkError(#[from] LinkError),
+
+    #[error(transparent)]
+    NoifyError(#[from] NotifyStreamError),
```

Comment:
This error case needs to be moved elsewhere.

`RuntimeError` must be independent of any specific feature (otherwise this error type will grow over time).

You can wrap any error into a `RuntimeError` using the `RuntimeError::ActorError` case or even simply wrapping an error in a `Box`.

Reply:
fixed